addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -245,7 +245,7 @@
 				<width>200</width>
 				<height>200</height>
 				<aspectratio aligny="center">keep</aspectratio>
-				<texture>$INFO[Player.Art(thumb)]</texture>
+				<texture fallback="$INFO[Player.Art(thumb)]">$INFO[VideoPlayer.EpgEventIcon]</texture>
```

Comment:
Revert this. This has to be done in another way. As I wrote in the other PR, there must not be "VideoPlayer.EpgSomething" as VideoPlayer does not know about EPG. This is a PVR thing.